docs/ops.html

```diff
@@ -1214,9 +1214,9 @@ <h4 class="anchor-heading"><a id="multitenancy-more" class="anchor-link"></a><a
   </p>
 
 
-  <h3 class="anchor-heading"><a id="config" class="anchor-link"></a><a href="#config">6.5 Kafka Configuration</a></h3>
```

Comment:
Should we undo the changes in this file as the new section is in `configuration.html`? If so let's undo the changes in `toc.html` too.